python/py-pylru/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pylru
+version             1.1.0
+revision            0
+categories-append   devel
+platforms           darwin
+license             GPL-2
+
+python.versions     27 34 35 36
+
+maintainers         @ipglider openmaintainer
+
+description         A least recently used (LRU) cache for Python.
+long_description    Pylru implements a true LRU cache along with several support classes.
+
+homepage            https://github.com/jlhutch/pylru
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
```

Comment:
I'm relatively ignorant of how the python portgroup works, so someone else should note if there's a simpler way to do this.

Reply:
The Python port group does not specify `master_sites`. As this pattern is quite common, it's not a bad idea to include it in the python-1.0 group IMO.

Reply:
@yan12125 If you could do a pull request for that, it would be appreciated!

Reply:
Let me try it out :D

Reply:
Unfortunately my Tcl knowledge is insufficient to make such an improvement. I've opened https://trac.macports.org/ticket/56642 as a record.